components/Dialogs/examples/DialogsCustomShadowExampleViewController.swift

```diff
@@ -42,12 +42,23 @@ class CustomShadowViewController: UIViewController {
     bodyLabel.sizeToFit()
     self.view.addSubview(bodyLabel)
 
+    self.view.isAccessibilityElement = true
+    self.view.accessibilityTraits = .button
+    self.view.accessibilityLabel = bodyLabel.text
+    self.view.accessibilityHint = "Dismiss this dialog"
+    let tap = UITapGestureRecognizer(target: self, action: #selector(dismissSelf))
```

Comment:
Can we not use the z-gesture here?

Reply:
This solves both the VoiceOver and non-VoiceOver use case in one go, because before this PR users also couldn't dismiss this dialog by tapping it (which was somewhat confusing).